rekolekcje-engine/src/main/java/pl/oaza/warszawa/dor/rekolekcje/api/parish/domain/Parish.java

```diff
@@ -1,14 +1,13 @@
 package pl.oaza.warszawa.dor.rekolekcje.api.parish.domain;
 
-import lombok.AllArgsConstructor;
-import lombok.Builder;
-import lombok.NoArgsConstructor;
-import pl.oaza.warszawa.dor.rekolekcje.api.parish.dto.ParishDTO;
-
 import javax.persistence.Entity;
 import javax.persistence.GeneratedValue;
 import javax.persistence.GenerationType;
 import javax.persistence.Id;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.NoArgsConstructor;
```

Comment:
\[Checkstyle\] WARNING: Import statement for 'lombok\.NoArgsConstructor' is in the wrong order\. Should be in the 'THIRD\_PARTY\_PACKAGE' group, expecting not assigned imports on this line\.